docs/headlampFAQ.md

```diff
@@ -0,0 +1,132 @@
+
+**A:** Headlamp is an open source, extensible web interface designed to simplify the management of Kubernetes clusters. It serves as a visual management tool ideal for developers, DevOps professionals, and system administrators who are looking for an intuitive way to oversee cluster operations, deployments, and troubleshooting.
+
+### Q: On which platforms can I run Headlamp?
+
+**A:** Headlamp is platform-agnostic in terms of its web interface, which can be accessed through any modern web browser. For desktop use, there are specific versions tailored for Windows, macOS, and Linux, ensuring a wide range of compatibility for different users.
+
+### Q: How can I get Headlamp up and running on my system?
+
+**A:** To install Headlamp, follow the detailed instructions provided in the [official installation guide](https://www.headlamp.dev/docs/latest/installation/). The process will guide you through downloading the application, setting up your Kubernetes cluster access, and launching Headlamp to connect to your cluster. For desktop applications, you can find additional information in the [desktop installation guide](https://www.headlamp.dev/docs/latest/installation/desktop/).
+
+### Q: Is there any cost involved in using Headlamp?
+
+**A:** Headlamp is completely free to use. As an open source tool under the Apache 2.0 License, it is available at no charge, and users are encouraged to modify and redistribute it in accordance with the license terms.
+
+### Q: Can I use Headlamp for commercial purposes?
+
+**A:** Yes, Headlamp is licensed for both personal and commercial use. The Apache 2.0 License allows for the use of Headlamp in any setting, from individual projects to enterprise-level operations.
+
+### Q: Where can I find the source code for Headlamp?
+
+**A:** The source code for Headlamp is publicly available on [GitHub](https://github.com/headlamp-k8s/headlamp). You are welcome to explore, fork, and contribute to the codebase.
+
+### Q: Who maintains Headlamp, and how is it funded?
+
+**A:** Headlamp is currently maintained by the original Kinvolk team, now part of Microsoft. The integration has provided the team with access to greater resources, enhancing their ability to develop, maintain, and expand Headlamp's capabilities.
+
+### Q: What features does Headlamp offer?
+
+**A:** Headlamp provides a range of features designed to enhance the Kubernetes experience. These include a clean and informative dashboard, the ability to inspect and edit Kubernetes resources, real-time cluster metrics, and plugin support for extensibility.
+
+### Q: How does Headlamp ensure the security of my Kubernetes cluster?
+
+**A:** Headlamp prioritizes security by using your local kubeconfig file to communicate directly with your cluster, ensuring that access control is governed by your Kubernetes configurations and credentials. It does not store your credentials on any server, minimizing the risk of unauthorized access.

```

Comment:
Let make this one more like:
Q: What capabilities/credentials does Headlamp require to access my Kubernetes cluster?
A: Headlamp doesn't require access to the cluster in itself. It instead relies to RBAC when connecting to the Kubernetes API server, meaning that it's the users that are required the credentials to access the cluster (via a service token or client certificate). Headlamp may store the token in the browser's local storage, but never in its backend/server.